endpoints.json

```diff
@@ -23,5 +23,58 @@
         }
       ]
     }
+  },
+  "GET /api/articles:article_id" : {
+    "description" : "serves an article object",
+    "queries" : [],
+    "example" : {"article": {    
+     " article_id": 1,
+      "title": "Living in the shadow of a great man",
+      "topic": "mitch",
+      "author": "butter_bridge",
+      "body": "I find this existence challenging",
+      "created_at": "2020-07-09T20:11:00.000Z",
+      "votes": "100"
+    }}
+  },
+  "PATCH /api/articles:article_id" : {
+    "description" : "serves an article object",
```

Comment:
I think you could be more descriptive here, it doesnt just serve an article object, it serves and update article request. Also you haven't given an example of the kind of request to send for this. Very important when we have patch/post/put requests as the client needs to know exactly how to send the data